packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,8 +10,17 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
-* [**] The media upload options of the Image, Video and Gallery block automatically opens when the respective block is inserted. [#29546]
-* [*] Image block: Improve text entry for long alt text. [#29670]
+-   [**] The media upload options of the Image, Video and Gallery block automatically opens when the respective block is inserted. [#29546]
+-   [*] Image block: Improve text entry for long alt text. [#29670]
+
+## 1.50.0
+
```

Comment:
Looks like this part is duplicated as per line `25` so I think we could remove these lines.